test/coreneuron/test_pointer.py

```diff
@@ -261,7 +268,8 @@ def run(tstop):
     from neuron import coreneuron
 
     coreneuron.enable = True
-    for perm in [0, 1]:
+    cell_permute_values = (1, 2) if coreneuron.gpu else (0, 1)
```

Comment:
Wondering if we should expose https://github.com/neuronsimulator/nrn/blob/7d905a3c17226a1b3a4af1430822c803d7d8e86b/share/lib/python/neuron/coreneuron.py#L41-L42 somehow instead of duplicating this logic in too many places 🤔

Reply:
IIUC, done in 6c38ad0900a4fec979c2f3d23eb09c14a4de802a.